jsk_2016_01_baxter_apc/StowWorkOrder[] array
